app.js

```diff
@@ -1,10 +1,16 @@
 const express = require('express')
 const app = express()
-const {getTopics} = require('./controllers/controllers')
+const {getTopics} = require('./controllers/topics-controller')
+const description = require('./endpoints.json')
+
 
 app.use(express.json());
 
 app.get('/api/topics', getTopics)
 
+app.get('/api', (req, res, next) => {
+    res.status(200).send({description})
+})
```

Comment:
You want to minimise code in the `app.js` file, so you could extract this callback to a controller function and require it in